cmake/modules/OCApplyCommonSettings.cmake

```diff
@@ -0,0 +1,12 @@
+function(apply_common_settings targetNamme)
```

Comment:
I'd include `target` in the name. Also, you may want to just check if the target exists (sure, the command itself will likely fail, too, but this makes debugging easier).

Also, I'd prefer to have this refactoring on a separate commit.